plugins/ci/commands/create-symptom.md

```
@@ -0,0 +1,83 @@
+---
+description: Interactively create (or update/delete) a Sippy Symptom that auto-labels a known CI failure pattern
+argument-hint: "[description-of-failure]"
+---
+
+## Name
+
+ci:create-symptom
+
+## Synopsis
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)